### ISSUE TITLE
Database Exception – yii\db\Exception

### ISSUE THREAD

**Issue by jasonmccallister** [putyourlightson/craft-campaign#186]
### Describe the bug
I'm getting the following error when using Campaign on PostgreSQL.

```
SQLSTATE[42703]: Undefined column: 7 ERROR: column "lastsent" does not exist
LINE 5: LEFT JOIN (SELECT "campaignId", MAX(lastSent) AS "lastSent"
```

### To reproduce
Steps to reproduce the behaviour:
1. Go to "/admin/campaign/campaigns"
3. See error message below:

```
SQLSTATE[42703]: Undefined column: 7 ERROR: column "lastsent" does not exist
LINE 5: LEFT JOIN (SELECT "campaignId", MAX(lastSent) AS "lastSent"
^
HINT: Perhaps you meant to reference the column "campaign_sendouts.lastSent".
The SQL being executed was: SELECT EXISTS(SELECT "elements"."id", "elements"."fieldLayoutId", "elements"."uid", "elements"."enabled", "elements"."archived", "elements"."dateCreated", "elements"."dateUpdated", "elements_sites"."id" AS "siteSettingsId", "elements_sites"."slug", "elements_sites"."siteId", "elements_sites"."uri", "elements_sites"."enabled" AS "enabledForSite", "campaign_campaigns"."campaignTypeId", "campaign_campaigns"."recipients", "campaign_campaigns"."opened", "campaign_campaigns"."clicked", "campaign_campaigns"."opens", "campaign_campaigns"."clicks", "campaign_campaigns"."unsubscribed", "campaign_campaigns"."bounced", "campaign_campaigns"."complained", "campaign_campaigns"."dateClosed", "lastSent", "content"."id" AS "contentId", "content"."title", "elements"."draftId", "drafts"."sourceId" AS "draftSourceId", "drafts"."creatorId" AS "draftCreatorId", "drafts"."name" AS "draftName", "drafts"."notes" AS "draftNotes", "drafts"."trackChanges" AS "draftTrackChanges", "drafts"."dateLastMerged" AS "draftDateLastMerged"
FROM (SELECT "campaign_sendouts"."lastSent" AS "lastSent", "elements"."id" AS "elementsId", "elements_sites"."id" AS "elementsSitesId", "content"."id" AS "contentId"
FROM "elements" "elements"
INNER JOIN "campaign_campaigns" "campaign_campaigns" ON "campaign_campaigns"."id" = "elements"."id"
LEFT JOIN (SELECT "campaignId", MAX(lastSent) AS "lastSent"
FROM "campaign_sendouts"
GROUP BY "campaignId") "campaign_sendouts" ON "campaign_sendouts"."campaignId" = "campaign_campaigns"."id"
INNER JOIN "campaign_campaigntypes" "campaign_campaigntypes" ON "campaign_campaigntypes"."id" = "campaign_campaigns"."campaignTypeId"
INNER JOIN "sites" "sites" ON "sites"."id" = "campaign_campaigntypes"."siteId"
INNER JOIN "elements_sites" "elements_sites" ON "elements_sites"."elementId" = "elements"."id"
INNER JOIN "content" "content" ON "content"."elementId" = "elements"."id"
INNER JOIN "drafts" "drafts" ON "drafts"."id" = "elements"."draftId"
WHERE ("sites"."dateDeleted" IS NULL) AND ("elements"."archived"=FALSE) AND ("elements"."dateDeleted" IS NULL) AND ("drafts"."sourceId" IS NULL) AND ("elements"."revisionId" IS NULL)
ORDER BY "elements"."dateCreated" DESC
LIMIT 1) "subquery"
INNER JOIN "campaign_campaigns" "campaign_campaigns" ON "campaign_campaigns"."id" = "subquery"."elementsId"
INNER JOIN "elements" "elements" ON "elements"."id" = "subquery"."elementsId"
INNER JOIN "elements_sites" "elements_sites" ON "elements_sites"."id" = "subquery"."elementsSitesId"
INNER JOIN "content" "content" ON "content"."id" = "subquery"."contentId"
INNER JOIN "drafts" "drafts" ON "drafts"."id" = "elements"."draftId"
ORDER BY "elements"."dateCreated" DESC)
```

### Expected behaviour
The page to show all my campaigns - which I do not have setup at the moment :)

### Screenshots
<img width="1417" alt="Screen Shot 2020-10-17 at 7 29 26 PM" src="https://user-images.githubusercontent.com/5354908/96355424-86186900-10af-11eb-8d8d-277341238037.png">

### Versions
- Plugin version: 1.16.3
- Craft version: 3.5.13.2

**Comment by bartroelands** [comment]
Have the same error - Craft version Craft Pro 3.5.13.2 / Plugin version 1.16.3 - Postgres 11.9

**Comment by bencroker** [comment]
Thanks for reporting, fixed in https://github.com/putyourlightson/craft-campaign/commit/68d49c05c948fa5183589e170a25525833c5308d and released in 1.16.4.
